pages/clustering/high-availability.mdx

```diff
@@ -361,10 +361,15 @@ the cluster enters a state of force reset where the cluster is reset to the stat
 
 The leader coordinator executes a force reset of the cluster if the action isn't fully complete. Failure can happen anywhere, i.e. in the case of setting instance to MAIN,
 the RPC request to a REPLICA instance to promote itself to MAIN can succeed, but writing to the Raft log that the instance was promoted can fail. 
-Force reset includes demoting every alive instance to REPLICA, and executing the failover procedure once again. Such a procedure is needed as currently cluster doesn't track 
+Force reset includes demoting every alive instance to REPLICA, and executing the failover procedure once again. Such a procedure is needed as of this moment cluster doesn't track 
 where the action failed exactly, but only whether it fully succeded. Raft log is taken as a source of truth at all times. In case the leader coordinator dies while executing 
 the force reset, the next coordinator which is elected as the leader, will continue executing the force reset. Action is executed until it succeeds.
 
+<Callout type="info">
+It is important to note that in case some action fails and all instances are down, the leader will try to execute force reset until one instance is promoted to MAIN. Until then no other action is allowed
+on cluster.
+</Callout>
```

Comment:
Replace text in Callout with:

It is important to note that if an action fails and all instances are down, the leader will attempt to execute a force reset until one instance is promoted to MAIN. Until then, no other actions are allowed on the cluster.

Add one blank line above and below this text (within Callout tags).